src/test/run-make/msvc-data-only/bar.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+extern crate foo;
+
+fn main() {
+    println!("The answer is {} !", foo::FOO);
```

Comment:
Could this be a `run-pass` instead of a `run-make` test? Without any C dependencies it should be fine for inclusion there. You'll need to tag the dependency as `// no-prefer-dynamic` as well as `#![crate_type = "rlib"]` to ensure it's not built as a DLL, hoewver.